index.html

```diff
@@ -329,6 +329,7 @@ <h3>
 <li><a href="https://github.com/ngx-cache/core">ngx-cache/core: Cache utility for Angular (incl. browser & server platform implementations) by</a><a href="https://github.com/fulls1z3" class="user-mention">@fulls1z3</a></li>
 <li><a href="https://github.com/ngx-i18n-router/core">ngx-i18n-router/core: Route internationalization utility for Angular by by</a><a href="https://github.com/fulls1z3" class="user-mention">@fulls1z3</a></li>
 <li><a href="https://github.com/SQ-UI/ng-sq-ui">NG-SQ-UI - Simple Quality UI kit for Angular. Build by developers for developers!</a></li>
+<li><a href="https://github.com/Sayegh7/ngx-wheel">ngx-wheel - Angular prize winning wheel component!</a></li>
```

Comment:
I think this is irrelevant at the moment.

Reply:
I wasn't sure if you guys needed it or not. I'll remove it.